src/components/Playground/Playground.tsx

```
@@ -64,220 +63,207 @@ const PlaygroundComponent: React.FC<PlaygroundComponentProps> = ({
       }
     }
-  }, [data])
+  }, [data]);
 
   // Create Loading component
-  const LoadingBar: CustomRenderBarProps = ({
-    content,
-    displayContent,
-    onSend
-  }) => {
+  const LoadingBar: CustomRenderBarProps = ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)